andi/andi.py

```diff
@@ -307,51 +313,87 @@ class we want to override. If ``recursive_overrides`` is True, then
                     full_final_kwargs=full_final_kwargs,
                     dependency_stack=None,
                     overrides=overrides,
-                    recursive_overrides=recursive_overrides)
+                    recursive_overrides=recursive_overrides,
+                    custom_builder_fn=custom_builder_fn,
+                    )
     return plan
 
 
+CustomBuilder = namedtuple("CustomBuilder", ["result_class_or_fn", "factory"])
```

Comment:
What do you think about using a dataclass instead?

Reply:
No 3.5 support

Reply:
I think we can drop 3.5

Reply:
And probably 3.6 and 3.7 as well, as we did in other packages

Reply:
Done